spec/dummy/app/controllers/projects_controller.rb

```diff
@@ -25,21 +25,28 @@ def update
 
   private
 
+  # def project_params
+  # 	params.require(:project).permit(
+  #     :title, :description,
+  #     tasks_attributes: [
+  #       :id, 
+  #       :title, 
+  #       :description, 
+  #       :_destroy, 
+  #       comments_attributes: [
+  #         :id, 
+  #         :content, 
+  #         :_destroy
+  #       ]
+  #     ],
+  #     participants_attributes: [:id, :email, :_destroy]
+  #   )
+  # end
+
+  # TODO: abstract the call to Klass.abyme_attributes inside a controller method
   def project_params
-  	params.require(:project).permit(
-      :title, :description,
-      tasks_attributes: [
-        :id, 
-        :title, 
-        :description, 
-        :_destroy, 
-        comments_attributes: [
-          :id, 
-          :content, 
-          :_destroy
-        ]
-      ],
-      participants_attributes: [:id, :email, :_destroy]
+    params.require(:project).permit(
+      :title, :description, Project.abyme_attributes
```

Comment:
could be cool to have something like:
```suggestion
      :title, :description, abyme_attributes
```

Reply:
C'est un one shot !